FtcRobotController/build.gradle

```diff
@@ -25,4 +25,10 @@ repositories {
     }
 }
 
+repositories {
+    flatDir {
+        dirs 'C:\\Users\\pston\\GMR\\GMR2016-2017\\RelicRecovery\\navx-micro\\navx-micro\\android\\libs'
```

Comment:
#AnswerPls Is there a way to change this so that it is a relative path? This path will have to be changed on everyone's individual computer.

Reply:
https://stackoverflow.com/questions/32079022/gradle-absolute-relative-path